rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |      N/A       |     11.0.0    |      11.0.0*   |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Ignition    |    Citadel*    |      N/A       |            Citadel*            |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     3.4.6*     |     4.2.0     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| OpenSSL     |     1.1.1d     |     1.1.1f     |     1.1.1f    |      1.1.1d    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.8.0*     |     1.9.0     |      1.9.0     |        1.9.4        |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.8.0      |     3.8.2     |      3.7.3     |  3.8.2 / 3.7.5****  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Qt          |     5.12.5     |     5.10.0     |     5.12.3    |      5.11.3    | 5.14.1 / 5.12.5**** |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+|                              |         **Linux only**                                                |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| PCL         |     1.10.0     |              N/A               |      1.9.1     |       1.10.0        |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+|                                **RMW DDS Middleware Providers**                                      |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Cyclone DDS |                                    0.7.x (Coquette)                                    |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Fast-DDS    |                                      2.1.x                                             |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Connext DDS |                              5.3.1              |               N/A                    |
```

Comment:
Will Connext 6 be an option? If so, maybe it's worth listing both versions

Reply:
Unclear at this time.  Right now, all we have in the default is 5.3.1, so I'd rather go with what we know for sure will work.  If Connext 6 becomes an option, we can update the table later.